.env

```diff
@@ -67,6 +67,7 @@ R_ORG=rhub
 R_IMAGE=ubuntu-gcc-release
 R_TAG=latest
 TZ=UTC
+R_STATIC_DEPENDENCY_SOURCE=BUNDLED
```

Comment:
I don't think we need this here. We have to put env vars here that affect what the Dockerfiles build, but this is a runtime env var, so we can just pass it with `-e` and don't need to declare it here.

Reply:
To add to that, here's one example of how that's done for a different CI job: https://github.com/apache/arrow/blob/master/.github/workflows/r-without-arrow.yml#L76

Reply:
And for here, I would do ~the opposite of what I said yesterday, we want something like `-e ARROW_DEPENDENCY_SOURCE=...` in the docker call since that's the environment variable that we advertise and use throughout Arrow to determine that source.

Yesterday I mentioned calling it something like `R_STATIC_DEP...` in the `.env` file, which if we needed to pass it there would be good to mark that it only was getting passed to the R builds and wasn't a parameter that would help on a python build. But Neal's right that we should just put it in the docker command in the GHA workflow file and not worry about passing it around in docker-compose or the .env file. Sorry for my misdirection there yesterday!